File: Pemro1_Modul6/Bahasa_C/PRAK604-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>
#include <string.h>

int main() {
    char kode[200], pesan[200];
    char hasil[200];
    int bintang = 0, pagar = 0;

    fgets(kode, sizeof(kode), stdin);
    fgets(pesan, sizeof(pesan), stdin);

    kode[strcspn(kode, "\n")] = '\0';
    pesan[strcspn(pesan, "\n")] = '\0';

    if (strlen(kode) != strlen(pesan)) {
        printf("Panjang kalimat berbeda, pesan palsu\n");
        return 0;
    }

    for (int i = 0; i < strlen(kode); i++) {
        if (kode[i] == pesan[i]) {
            hasil[i] = '*';
            bintang++;
        } else {
            hasil[i] = '#';
            pagar++;
        }
    }

    hasil[strlen(kode)] = '\0';

    printf("%s\n", hasil);
    printf("* = %d\n", bintang);
    printf("# = %d\n", pagar);

    if (bintang >= pagar)
        printf("Pesan Asli\n");
    else
        printf("Pesan Palsu\n");

    return 0;
}

File: Pemro1_Modul6/Bahasa_Phyton/PRAK604-2510817320010-PutriDeissytaHerliyana.py
kode = input()
pesan = input()

if len(kode) != len(pesan):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    hasil = ""
    bintang = 0
    pagar = 0

    for i in range(len(kode)):
        if kode[i] == pesan[i]:
            hasil += "*"
            bintang += 1
        else:
            hasil += "#"
            pagar += 1

    print(hasil)
    print(f"* = {bintang}")
    print(f"# = {pagar}")

    if bintang >= pagar:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")
